tweets/2020/06/13x-next.tweet

```diff
@@ -0,0 +1,2 @@
+Nock's 13.x release is already live on next and close to being shipped on latest! The team would like outside feedback on the release.
+If you have a moment, please give it a try. https://github.com/nock/nock/releases/tag/v13.0.0
```

Comment:
I think I'd mention the npm command, and instead of "the team" I think I'd say "we" to make it more personal

```suggestion
📯 Nock's 13.x is about to be released! We would love your feedback before the release hits latest on npm.

https://github.com/nock/nock/releases/tag/v13.0.0

You can test the upcoming release today using:

npm install --save-dev nock@next
```